response.proto

```diff
@@ -330,7 +330,7 @@ message ScheduleStopTime {
     optional RTLevel realtime_level = 6;
     optional uint64 base_date_time = 7;
     repeated string impact_uris = 8;
-    optional int32 occupancy = 9; // percentage of the vehicle total capacity filled by travellers
+    optional string occupancy = 9; // occupancy status as described in https://gtfs.org/reference/realtime/v2/#enum-occupancystatus
 }
```

Comment:
Since occupancy is an enum, I suggest :

```suggestion
    optional Occupancy occupancy = 9; // occupancy status as described in https://gtfs.org/reference/realtime/v2/#enum-occupancystatus
}

enum Occupancy {
EMPTY =0;
MANY_SEATS_AVAILABLE = 1;
FEW_SEATS_AVAILABLE = 2;
STANDING_ROOM_ONLY = 3;
CRUSHED_STANDING_ROOM_ONLY = 4;
FULL = 5;
NOT_ACCEPTING_PASSENGERS = 6;
}
```

Reply:
As decided we wont make any transformation and use the text value send by forseti.

Reply:
allright !